src/plugin.hpp

```diff
@@ -4,9 +4,9 @@
 #include <Core/CoreMacros.hpp>
 /// Defines the correct macro to export dll symbols.
 #if defined  DPluginExample_EXPORTS
-    #define PLUGIN_EXAMPLE_API DLL_EXPORT
+    #define PLUGIN_API DLL_EXPORT
```

Comment:
Are you sure about this ? I think we need a different macro per plugin and `PLUGIN_API` is a bit too generic, no ?

Reply:
Well, I'm not really aware of these window-centric stuff. For me, as each plugin source live in separate world and must implement a interface given by Radium, these macro must also be provided by Radium and not chosen arbitrarily by the plugin developer.
Nevertheless, I made the requested modifications.

Reply:
@nmellado what do you think about that ?